pkg/apiserver/crud/handler.go

```diff
@@ -55,3 +56,15 @@ func getApiViewFromHeader(reqHeaders http.Header) string {
 
 	return DefaultApiView
 }
+
+const mysqlDuplicateEntryError = 1062
+
+func IsDuplicateEntryError(err error) bool {
```

Comment:
I am not sure if this really belongs in the apiserver/crud namespace. I would rather see it in the db namespace.